spec/features/admin_can_add_categories_to_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.